ChustaSoft.Tools.Authorization/Helpers/SecurityKeyHelper.cs

```diff
@@ -8,10 +8,13 @@ public static class SecurityKeyHelper
 
         public static SecurityKey GetSecurityKey(string privateKey)
         {
-            var secretKeyBytes = Encoding.UTF8.GetBytes(privateKey);
-            var signingKey = new SymmetricSecurityKey(secretKeyBytes);
+            if (!string.IsNullOrEmpty(privateKey)) 
```

Comment:
Remove this validation. 
If private key is not setted up, should throw an Exception because it is mandatory for the right behaviour of the tool